assets/dev/js/frontend/handlers/base-nested-tabs.js

```diff
@@ -253,12 +253,12 @@ export default class BaseNestedTabs extends Base {
 	changeActiveTab( tabIndex, fromUser = false ) {
 		// `document/repeater/select` is used only in edit mod, and only when its not internal call,
 		// in other words only in editor and when user triggered the change.
-		if ( fromUser && this.isEdit ) {
-			return window.top.$e.run( 'document/repeater/select', {
-				container: elementor.getContainer( this.$element.attr( 'data-id' ) ),
-				index: parseInt( tabIndex ),
-			} );
-		}
+		// if ( fromUser && this.isEdit ) {
+		// 	return window.top.$e.run( 'document/repeater/select', {
+		// 		container: elementor.getContainer( this.$element.attr( 'data-id' ) ),
+		// 		index: parseInt( tabIndex ),
+		// 	} );
+		// }
 
 		const isActiveTab = this.isActiveTab( tabIndex ),
 			settings = this.getSettings();
```

Comment:
Do these values `settings.toggleSelf` and `settings.hidePrevious` ever changes for Nested Tabs? In other words, would it make a difference if we would remove them?